Feature branch-Merge

aracelicaes opened the pull request:
We are creating the pull request for the feature branch to go into the master given approval.

aracelicaes committed:
Added linters

Added js linter and yml linter manualy to repo

aracelicaes committed:
Created HTML and CSS files

Created HTML and CSS files and uploaded

aracelicaes committed:
Added HTML structure

Added HTML structure, DOCTYPE, html, head, body, main, nav, form, fieldset, footer.

aracelicaes committed:
Created IMAGES folder

Created images folder and inserted 3 png images and 1 logo JPG

HeyItsGwen committed:
Style navbar

Styled the navbar with sign in button, and lost password information. Gave button similar background and font color to replicate the site.

aracelicaes committed:
added header and content

Added header along with Intuit logo and other logo+company names and descriptions.

aracelicaes committed:
Styled Header, Images and Text inside

Added stylings, centered and aligned content inside the header.

HeyItsGwen committed:
Added inputs

added inputs for email phone number and password, as well as submit button, and styled the form and fieldset to make it look nicer.

HeyItsGwen committed:
Add info text

Added info text and learn more link inside the form. Changed input styles to make it more centered, and centered text within the form.

HeyItsGwen committed:
Made info margins

Added margins to make the form elements more correctly match the example given.

aracelicaes committed:
Added stylings

added stylings to phoneInfo, labels, and infoDiv
Also changed size of input boxes.

aracelicaes committed:
styled button

Styled Create Account button, added color padding radius

aracelicaes committed:
added larger Icons

Added larger Icons and upated HTML code with larger icons

aracelicaes committed:
Added Disclosure and styled it

Added disclosure satement and styled it!

HeyItsGwen committed:
added legal text

Added legal text to the footer and styled it to match the example given.

HeyItsGwen committed:
Validated HTML/CSS

Validated and fixed the HTML/CSS using w3 tools.

aracelicaes committed:
Updated README

Updated Description details for the README file.

ebukaume commented:
## Changes Required! :stop_sign: 

Hi @aracelicaes  and @HeyItsGwen :wave: 

You guys have really done a good job here. The clone looks good. 

Let's fix these issues listed below so we can move forward.

- [x] Make linter (stickler) run on this repo. You can refer to [this guide](https://github.com/microverseinc/linters-config/tree/master/css) to make sure you are not missing something in the setup.

- [ ] The project should have a descriptive README.md ( _You can refer to [the guide](https://github.com/othneildrew/Best-README-Template) for inspiration on how to craft an awesome README_ )

- [x] You should move your navbar out of the `<main>` tag so it has this `<nav><main><footer>` transistion

**Kindly submit another code review request after fixing the above issues.**


Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

aracelicaes committed:
Fixed Feedback Error for NAV

Took the Nav out of the MAIN element and put it outside but under  <body> according to project feedback.

HeyItsGwen committed:
Reupload linters

Trying to make stickler work, reuploading stickler and stylint config to repo

HeyItsGwen committed:
Fixing some stickler errors

Fixing the space issue that linter gave after it's check

HeyItsGwen committed:
fixing more lint style issues

Fixed hex code capitolization and more spacing

HeyItsGwen committed:
Expected indentation

Stickler says expected indentation of 2 spaces and I'm trying to figure out what that means

HeyItsGwen committed:
Still indentation

Still haven't figured out what that means

HeyItsGwen committed:
More indentation

Think I figured out what it means, but if I'm right it means don't tab each css property forward but use 2 spaces which is weird

aracelicaes committed:
Fixed Style Linter error

Style linter on GITHUB said to reduce from 4 tab spaces to 2, so we are fixing this and reducing tab spaces from 4 to 2.

aracelicaes committed:
Stickler Fix

Stickler said to add 0 before .8 rem so did that for all. Also said to take footer a and insert it BEFORE #infoDiv. Fixed that as well.

aracelicaes committed:
Stickler Styling Error

Fixed Stickler Styling Error
Expected selector "header div a" to come before selector "footer a:hover" (no-descending-specificity) [error

HeyItsGwen committed:
Testing README

Added links to the README and pushing to test

HeyItsGwen committed:
Updating README

Updated the README to match the specifications set in our code review.

aracelicaes committed:
added screenshot of project

added screenshot of project

HeyItsGwen committed:
Add screenshot image

Adding the screenshot image to the link

ebukaume commented:
## Approved! :champagne: 

Hi @aracelicaes  and @HeyItsGwen :wave: 

**Congratulations! :fireworks: You guys have really done well. You can go ahead and merge this branch.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)
